### DISCUSSION TITLE
feat: can create customized channels to store colorful data

### DISCUSSION THREAD

**Discussion by madoka-chann** [Samarium150/mirai-console-drift-bottle#15] [Ideas]
> 不知道，不了解 ≈ 不存在（0）

无可厚非，人们通常对别人所处的境遇呈现出冷漠的态度：我们不知道的人事纷纭，即使确切存在，我们也会理所当然的把他们当做不存在。鲜少有人对着素未谋面的网友表现出充分的热情和关怀；或去对他们写下的东西评头论足。这样获得的乐趣，至少相比于关系更近一点的人是廉价的。

总体而言，人们只会对自己身边的人、和能让自己获得充足乐趣的事感到在意。

> 越近，越在意。

所以，如果只能面向所有人扔漂流瓶、捡漂流瓶，群友们通常都是在面临着上面所讲的趋近于【不存在】的情况。虽说的确会有人对和自己距离很远的人事有着强烈的执着，但其影响，对增加【本插件】对【所有使用者】【使用本插件】的【乐趣】而言 可以忽略不记。

问几个问题：
1、你舍友和班里一个女同学有关系了，你会感兴趣吗？
2、你隔壁班的某个男生和他班里的女生谈恋爱了，你会感兴趣吗？
3、上面两个问题，你更感兴趣哪个？

如果说某群群友扔出的【漂流瓶】可以限制在【自己所在】的群内传播，那么拉近人与人之间关系的【容器】就是【群本身】，催化剂便是【群漂流瓶】，而扔【群漂流瓶】就是条件。在初中的化学课上，我们就已经了解到到催化剂让化学产业【工业化】成为可能，可见催化剂其力量之所在。

综上所述，本插件可以通过增加【群漂流瓶】的分支，让本插件の乐趣和哲学得到【质的飞跃】。

**Comment by madoka-chann** [comment]
引申出来了一个idea：

漂流瓶所处的环境可以被定义为【频道】
（或者形象的称之为社区留言板）
如果说目前漂流瓶都处在【世界频道】
那么，我上面所提出的环境就可以被称为【群频道】

那么问题来了，假如能够由用户自创频道呢？如果所有用户都能自创频道呢？！
嘛，想多了想多了。比较简洁明了的方法就是直接在配置文件加就行了，然后再通过指令远程实现。

Perfect (≧O≦)〃

不行，还得想想，如果可以呢？
大概可以通过以下：

1、全自助审核

如果全部qq用户都有权限创建频道，那么必须要通过一种方式收集他们所给出的【频道类型】和【频道名】，这些数据需要存储在云端，后来者可以直接【跟票】。超级管理员可以手动同意某个频道，或者某个频道达到一定的票数自动创建。

2、人工审核

用户在群内申请创建某个【漂流瓶频道】，机器人pass给超管，同意或拒绝，返回结果给用户（群内@）

关于查重：用一些匹配表达式就行了。如果用户在申请的时候频道名和已有部分重叠，返回重叠的频道名给用户。若用户选择继续创建，机器人pass给超管...

——————————
先想这么多吧，主要还是做一个想法的总结和引申。频道这个概念出来了就行，应该可以搞一些好玩的小东西在上面。

**Comment by madoka-chann** [comment]
这个data可以是colorful的，是后续可以拓展的。最基本的肯定就是和漂流瓶有关的数据
细化：

漂流瓶可以附加属性，附加...

漂流瓶附加属性和、附加...可以产生什么效果，触发什么事件、事件在谁身上触发、是单向还是双向还是随机

将漂流瓶数据和用户数据联系起来...

反正漂流瓶这个玩法真的可以有很多思路，棒棒哒！目前最吸引我的一个插件~~~前途无限 ٩(^ᴗ^)۶

 incomplete ideas, waiting to add more thoughts：
极速漂流，定时漂流，污染漂流，vip漂流，永恒漂流，etc.

**Comment by madoka-chann** [comment]
[Mentioned this thing](https://github.com/FloatTech/ZeroBot-Plugin/issues/2#issuecomment-1038713035)

**Comment by U29kaXVt** [comment]
帮你简单翻译一下：
第一条：新增将漂流瓶数据按群号存储对应群的漂流瓶
第二条：漂流瓶中新增小分区
第三条：加一点吊炸天的特技
（（（（（

**Comment by LaoLittle** [comment]
给钱
